contents/stats_vis.md

```diff
@@ -1,7 +1,181 @@
 ## Statistical Visualizations {#sec:stats_vis}
 
+There are several statistical visualization techniques.
+We will focus on mainly two: **box plots** and **density plots**, since they are the most used and our preferred way to analyze univariate data.
+
+We will also use the same `more_grades` dataset from @sec:stats_central.
+
 ### Box Plots {#sec:stats_vis_boxplots}
 
+Box plots are a method for graphically depicting numerical data through their quartiles.
+The "box" is typically represented by the quartiles 1 to 3.
+The median, second quartile -- Q2, or percentile 0.5, is the line inside the box.
+The first and third quartile, Q1 and Q3, or percentiles 0.25 and 0.75, respectively, are the box's lower and upper bounds.
+Finally, we have the "whisker" which, traditionally (and default in most data visualization techniques), is the range composed by extending the interquartile range (IQR) by 1.5.
+
+The basic box plot can be drawn using `Makie.jl` (see Chapter -@sec:datavisMakie).
+It accepts a `x` and `y` vectors which represents the positions of the categories and the variables within the boxes, respectively.
+Since our vector `x` is a `Vector{String}` we need to convert it to `categorical` using `CategoricalArrays` (@sec:missing_data) and then pass the `Axis` keyword argument `xticks` (see @sec:datavisMakie_attributes) as a tuple of values and labels.
```

Comment:
```suggestion
Since the elements in our vector `x` are of type `String`, we need to convert it to `categorical` using `CategoricalArrays.jl` (@sec:missing_data) and then pass the `Axis` keyword argument `xticks` (see @sec:datavisMakie_attributes) as a tuple of values and labels.
```